fabischapeli/fast-food-fast-v1
# #160231963 returns all the meals in the application

What does this PR do?

It returns all the meals present in the app

Description of Task to be completed?

The code should be able to return all meals  in json format when tested using postman

How should this be manually tested?

Fork this branch to your github and clone it to your machine.
Go to chrome, more tools, extensions and add postman
On postman, make sure the server is running and add the method to be tested plus the routes and click send
You should get the information in json format else there is a bug in your code or the server is not running

Any background context you want to provide?

The server is written using flask miniframework

What are the relevant pivotal tracker stories?

#160231963
